### PR TITLE
Module certification issues

### PR DESCRIPTION
**Pull request by anjalimukundan**

### CODE REVIEW

**Comment by neeradanelxsi** [comment]
Closing PR as it was created against main branch of FCS

File: cypress/support/cypress-commands/lifecycle.js [hunk 1]
```diff
@@ -293,7 +293,19 @@ Cypress.Commands.add('invokeLifecycleApi', (appId, method, methodParams = null)
...
+    publishMessageForLog = publishMessage.data.query;
+    if (typeof publishMessageForLog == 'string') {
+      publishMessageForLog = JSON.parse(publishMessageForLog);
+    }
+    if (publishMessageForLog.params.methodName == 'Lifecycle.history') {
+      cy.log(CONSTANTS.LIFECYCLE_HISTORY_INTENT + JSON.stringify(publishMessage));
+    } else {
+      cy.log(CONSTANTS.LIFECYCLE_INTENT + JSON.stringify(publishMessage));
+    }
+  }
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/lifecycle.js:308]: cant we use this list 
CONSTANTS.LIFECYCLE_APIS to determine if its a lifecycle api or history ?

Comment (anjalimukundan) [reply cypress/support/cypress-commands/lifecycle.js:308]: updated to     if (publishMessageForLog.params.methodName == CONSTANTS.LIFECYCLE_APIS.HISTORY) {

**Review by neeradanelxsi — DISMISSED** [review]

**Review by jbigel — APPROVED** [review]

File: cypress/fixtures/objects/errorContentObjects.json [hunk 2]
```diff
@@ -10,5 +10,9 @@
     },
     "INVALID_TYPE_PARAMS": {
         "type": "schemaOnly"
```
Comment (neeradanelxsi) [inline cypress/fixtures/objects/errorContentObjects.json:12]: you can remove INVALID_TYPE_PARAMS

File: cypress/TestCases/FireboltCertification/Advertising.feature [hunk 3]
```diff
@@ -113,9 +113,8 @@ Feature: Advertising
    @sdk @transport
    Scenario Outline: Advertising.advertisingId - Positive Scenario: <Scenario>
       When 1st party app invokes the 'Firebolt' API to '<MANAGE_API_Key>'
-      And '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
       Then 'Firebolt' platform responds to '1st party app' with '<MANAGE_API_Key>'
-      When '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
+      And '3rd party app' invokes the 'Firebolt' API to '<CORE_API_Key>'
```
Comment (jmeier204) [inline cypress/TestCases/FireboltCertification/Advertising.feature:117]: This should be 'When' keyword. Do. not understand why it was changed to 'And'

File: cypress/fixtures/objects/errorContentObjects.js [hunk 4]
```diff
@@ -27,3 +27,6 @@ exports.NOT_AVAILABLE = {
 exports.INVALID_TYPE_PARAMS = {
```
Comment (jmeier204) [inline cypress/fixtures/objects/errorContentObjects.js:27]: If we removed 'INVALID_TYPE_PARAMS' We can probably remove this Export Also.

**Review by jmeier204 — CHANGES_REQUESTED** [review]

File: cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature [hunk 5]
```diff
@@ -81,37 +81,37 @@ Feature: ClosedCaptions_Manage
...
+            | Set textAlign-true          | textAlign          | true       | INVALID_PARAMS |
+            | Set textAlignVertical-123   | textAlignVertical  | 123        | INVALID_PARAMS |
+            | Set textAlignVertical-true  | textAlignVertical  | true       | INVALID_PARAMS |
+            | Set fontColor-123           | fontColor          | 123        | INVALID_PARAMS |
+            | Set fontColor-true          | fontColor          | true       | INVALID_PARAMS |
+            | Set fontOpacity-120         | fontOpacity        | 120        | INVALID_PARAMS |
+            | Set fontOpacity-test        | fontOpacity        | test       | INVALID_PARAMS |
+            | Set fontOpacity-true        | fontOpacity        | true       | INVALID_PARAMS |
+            | Set backgroundOpacity-120   | backgroundOpacity  | 120        | INVALID_PARAMS |
+            | Set backgroundOpacity-test  | backgroundOpacity  | test       | INVALID_PARAMS |
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature:116]: if all of the values are same, why do we need a separate column here

File: cypress/TestCases/FireboltCertification/Advertising.feature (hunk 3 above)
Comment (ibrahimshuail) [reply cypress/TestCases/FireboltCertification/Advertising.feature]: review comment addresed

File: cypress/fixtures/objects/errorContentObjects.json (hunk 2 above)
Comment (ibrahimshuail) [reply cypress/fixtures/objects/errorContentObjects.json]: review comment addresed

File: cypress/fixtures/objects/errorContentObjects.js (hunk 4 above)
Comment (ibrahimshuail) [reply cypress/fixtures/objects/errorContentObjects.js]: review comment addresed

File: cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature (hunk 5 above)
Comment (ibrahimshuail) [reply cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature]: review comment addresed

**Review by jmeier204 — APPROVED** [review]

**Comment by anjalimukundan** [comment]
closed without merging

**Pull request closed without merging**